recipe/meta.yaml

```diff
@@ -41,6 +41,12 @@ outputs:
         - termcolor-cpp
         - cli11
         - libiconv                               # [unix]
+    test:
+      commands:
+        - test -e $PREFIX/lib/libmamba.so  # [linux]
```

Comment:
you can use `${SHLIB_EXT}`, e.g. https://github.com/conda-forge/nss-feedstock/blob/0b568c7e2d505eb8f1c3f55f06328588657318c7/recipe/meta.yaml#L41-L47

Reply:
done!